odl/solvers/nonsmooth/douglas_rachford.py

```diff
@@ -28,7 +28,7 @@
 __all__ = ('douglas_rachford_pd',)
 
 
-def douglas_rachford_pd(x, prox_f, prox_cc_g, L, tau, sigma, niter,
+def douglas_rachford_pd(x, f, g, L, tau, sigma, niter,
```

Comment:
I find it hard remembering the difference between `f`, `g`, `h`, `l` and the other stuff. If you need to consult the documentation every single time, the names are not good. Especially when you see the function "in the wild", you need to go back every time and check which name was which. 

And matching paper notions is secondary IMO. What if we find another variant of the method in another paper that uses different notions, and we want to put it into the same function, like in the case of memory limited BFGS?

I think it would be good to give at least some hint about the nature of the variables in the names. They typically differ in attributes like "smooth", "composed", "inf-convolved", which would set them apart quite nicely.